include/roaring/roaring_array.h

```diff
@@ -95,6 +95,14 @@ inline container_t *ra_get_container_at_index(
  */
 uint16_t ra_get_key_at_index(const roaring_array_t *ra, uint16_t i);
 
+/**
+ * Check whether value should be placed in the container at index i
+ */
+static inline bool ra_test_value_at_index(const roaring_array_t *ra, uint16_t i, uint32_t value) {
```

Comment:
Do we need this as a new function? It is a single line and it is not clear that it improves the quality of the code.

When someone is reading your code, they have to go look up this one function, and the function spans a single line.

And why is it coded this way? Don't you just want

```
ra_get_key_at_index(ra, i) == (uint16_t)(value >> 16)
```

Why the xor ?

Reply:
nothing special

fixed